google/style.css

```diff
@@ -97,11 +98,12 @@ square-dot {
   display: -ms-grid;
   display: grid;
   text-align: center;
+  height: 100%;
 }
 
 .search-bar {
-  width: 420px;
-  margin-top: 20px;
+  width: 520px;
```

Comment:
might be better to set your width in % as opposed to px. Can be better for sizing on different devices